packages/cli/src/util/paths.ts

```diff
@@ -0,0 +1,179 @@
+import { generateHttpParam } from '@stoplight/prism-http';
+import { serializeWithDeepObjectStyle } from '@stoplight/prism-http';
+import { serializeWithPipeDelimitedStyle, serializeWithSpaceDelimitedStyle } from '@stoplight/prism-http';
+import {
+  Dictionary,
+  HttpParamStyles,
+  IHttpOperation,
+  IHttpParam,
+  IHttpPathParam,
+  IHttpQueryParam,
+  INodeExample,
+  INodeExternalExample,
+} from '@stoplight/types';
+import * as Either from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { get } from 'lodash';
+// @ts-ignore
+import { parse } from 'uri-template';
+
+export function createExamplePath(operation: IHttpOperation): Either.Either<Error, string> {
+  return pipe(
+    generateTemplateAndValuesForPathParams(operation),
+    Either.chain(({ template: pathTemplate, values: pathValues }) => {
+      return pipe(
```

Comment:
Nitpick (you can ignore this) if you're just returning pipe, use a single statement, it improves the readability a lot.